calico/scripts/install-calico-windows.ps1

```diff
@@ -50,35 +50,31 @@ function DownloadFiles()
 
     Write-Host "Downloading Windows Kubernetes scripts"
     DownloadFile -Url  https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1 -Destination $BaseDir\hns.psm1
-    DownloadFile -Url  https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/InstallImages.ps1 -Destination $BaseDir\InstallImages.ps1
-    DownloadFile -Url  https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/Dockerfile -Destination $BaseDir\Dockerfile
-}
-
-function PrepareDockerFile()
-{
-    # Update Dockerfile for windows
-    $OSInfo = (Get-ComputerInfo  | select WindowsVersion, OsBuildNumber)
-    $OSNumber = $OSInfo.WindowsVersion
-    $ExistOSNumber = cat c:\k\Dockerfile | findstr.exe $OSNumber
-    if (!$ExistOSNumber)
-    {
-        Write-Host "Update dockerfile for $OSNumber"
-
-        $ImageWithOSNumber = "nanoserver:" + $OSNumber
-        (get-content c:\k\Dockerfile) | foreach-object {$_ -replace "nanoserver", "$ImageWithOSNumber"} | set-content c:\k\Dockerfile
-    }
 }
 
 function PrepareKubernetes()
 {
     DownloadFiles
-    PrepareDockerFile
     ipmo C:\k\hns.psm1
-
-    # Prepare POD infra Images
-    c:\k\InstallImages.ps1
-
     InstallK8sBinaries
+
+    # Prepull and tag the pause image for docker
+    if (-not (Get-IsContainerdRunning)) {
```

Comment:
Is pause image used by both docker and containerd? I'm not sure about we need `if not containd` here.

Reply:
The pause image specified in the kubelet args is only for Docker. For Containerd, this is specified by its own config.